test/metric_dimension/metrics_aggregate_dimension_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build linux && integration
+// +build linux,integration
```

Comment:
Nit: // +build having this is no longer recommened.

Reply:
ty! will take a look at reasons